cfg/std.cfg

```diff
@@ -7506,11 +7506,14 @@ initializer list (7) string& replace (const_iterator i1, const_iterator i2, init
     <alloc init="false" buffer-size="malloc">malloc</alloc>
     <alloc init="true" buffer-size="calloc">calloc</alloc>
     <alloc init="false" buffer-size="malloc:2">aligned_alloc</alloc>
+    <realloc init="false" buffer-size="malloc:2">realloc</realloc>
+    <realloc init="false" buffer-size="calloc:2,3">reallocarray</realloc>
```

Comment:
I think this needs to be `init="true"` to avoid uninit fps for code like 
```
    char* p = calloc(1, 10);
    p = realloc(p, 20);
    f(p[1]); // ok
    f(p[10]); // not ok but it's better to have a  FN than a  FP
```